code/datums/gamemode/role/vampire_role.dm

```diff
@@ -197,18 +199,29 @@
 		if(!target.vessel.get_reagent_amount(BLOOD))
 			to_chat(assailant, "<span class='warning'>They've got no blood left to give.</span>")
 			break
+		if (!(target in feeders))
+			feeders[target] = 0
```

Comment:
Would this be keeping a hard reference to the bitten?

Might cause trouble regarding garbage collection

Reply:
would doing feeders["/ref[target]"] fix it

Reply:
That would be a way of doing it. Would require a bit more sanity (checking that it's actually a human at that memory address, for instance)